web3-actions/priceFeedMonitor/tokenPairMonitor/btcUsd.ts

```diff
@@ -0,0 +1,26 @@
+import { ActionFn, Context, Event, PeriodicEvent } from "@tenderly/actions";
+import { checkTokenPairPriceUpdateTime } from "./validate";
+
+// Define the contract address and token pair
+const CONTRACT_ADDRESS = "0xb4e6A7861067674AC398a26DD73A3c524C602184";
```

Comment:
I'll change the address once a contract will be deployed.

Reply:
To avoid any mistakes, should we not always initially commit `CONTRACT_ADDRESS = ""`?